pkg/cap/cluster/container.go

```diff
@@ -0,0 +1,233 @@
+package cluster
+
+import (
+	"context"
+
+	"capnproto.org/go/capnp/v3"
+	"capnproto.org/go/capnp/v3/server"
+	api "github.com/wetware/ww/internal/api/cluster"
+	"github.com/wetware/ww/pkg/vat"
+)
+
+var (
+	ContainerCapability = vat.BasicCap{
```

Comment:
I don't think we need a separate capability for `Container` vs `Host`.  We should probably just have `AnchorCapability` with the paths `anchor` and `anchor/packed`.

(Note:  at some point, these paths are going to have to change anyway, to prevent clients from simply dialing to `anchor` and bypassing authentication.  But let's not worry about that for now.)

Reply:
The reason why I did this is because I thought it was not possible to export different capabilities under the same protocol. So, is it safe to do so?

Reply:
I think it is since we always know what we're getting back based on the `Walk` path.  Let's address this later though - it's good enough for now.